apps/tandoor/docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+version: "3.7"
+services:
+  tandoor:
+    container_name: tandoor
+    image: ghcr.io/tandoorrecipes/recipes:1.5.4
+    volumes:
+      - ${APP_DATA_DIR}/data/staticfiles:/opt/recipes/staticfiles
+      - ${APP_DATA_DIR}/data/mediafiles:/opt/recipes/mediafiles
+    ports:
+      - ${APP_PORT}:8080
+    environment:
+      - SECRET_KEY=${TANDOOR_SECRET_KEY}
+      - DB_ENGINE=django.db.backends.postgresql
+      - POSTGRES_HOST=tandoor_db
+      - POSTGRES_PORT=5432
+      - POSTGRES_USER=djangodb
+      - POSTGRES_PASSWORD=${TANDOOR_POSTGRESS_PASSWORD}
+      - POSTGRES_DB=djangodb
```

Comment:
Also for postgres db,"tandoor", it will also avoid conflict if overridden by user to use a unique postgres instance